File: Python/miniCalculator.py
print("Enter two numbers and operation: ")
a,b=  map(int,input().split())
print("Enter operation: ")
op = input()

if(op == '+'):
    print(f"Sum: {a+b}")
elif(op == '-'):
    print(f"Difference: {a-b}")
elif(op == '*'):
    print(f"Product: {a*b}")
elif(op == '/'):
    print(f"Divide: {a/b}")
elif(op == '%'):
    print(f"Modulus: {a%b}")
else:
    print("Invalid operation")

File: C++/miniCalculator.cpp
#include<iostream>
using namespace std;

int main(){
    int a,b;
    char op;
    cout<<"Enter two number and operation: ";
    cin>>a>>b>>op;

    switch (op)
    {
    case '+':
        cout<<"Sum: "<<a+b<<endl;
        break;
    case '-':
        cout<<"Difference: "<<a-b<<endl;
        break;
    case '*':
        cout<<"Product: "<<a*b<<endl;
        break;
    case '/':
        cout<<"Divide: "<<a/b<<endl;
        break;
    case '%':
        cout<<"Modulus: "<<a%b<<endl;
        break;
    
    default:
        cout<<"Invalid operation requested";
        break;
    }

}